config/webpack.prod.config.js

```diff
@@ -172,26 +194,10 @@ module.exports = merge(commonConfig, {
       path: path.resolve(process.cwd(), '.env'),
       systemvars: true,
     }),
-    new HtmlWebpackNewRelicPlugin({
-      // This plugin fixes an issue where the newrelic script will break if
-      //  not added directly to the HTML.
-      // We use non empty strings as defaults here to prevent errors for empty configs
-      accountID: process.env.NEW_RELIC_ACCOUNT_ID || 'undefined_account_id',
-      agentID: process.env.NEW_RELIC_AGENT_ID || 'undefined_agent_id',
-      trustKey: process.env.NEW_RELIC_TRUST_KEY || 'undefined_trust_key',
-      licenseKey: process.env.NEW_RELIC_LICENSE_KEY || 'undefined_license_key',
-      applicationID: process.env.NEW_RELIC_APP_ID || 'undefined_application_id',
-    }),
-    new NewRelicSourceMapPlugin({
-      applicationId: process.env.NEW_RELIC_APP_ID,
-      apiKey: process.env.NEW_RELIC_ADMIN_KEY,
-      staticAssetUrl: process.env.BASE_URL,
-      // upload source maps in prod builds only
-      noop: typeof process.env.NEW_RELIC_ADMIN_KEY === 'undefined',
-    }),
+  ].concat(extraPlugins).concat([
```

Comment:
Readability nit: Might suggest replacing this line with just:

```
...extraPlugins,
```

And leaving all the plugins in the same array.  The spread operator will just splat the extra ones into the array.  And I think we can also move `BundleAnalyzerPlugin` above the extra ones, which will help it feel a bit more organized.

Reply:
TIL the spread operator!